pipeline/build-rpm-package.yaml

```diff
@@ -92,13 +92,16 @@ spec:
         configuration, dist-git-client, koji-client, and other RPM-build related
         tooling.
       name: script-environment-image
-      default: quay.io/redhat-user-workloads/rpm-build-pipeline-tenant/environment:44db26f1861de8486478c1eb78bf2c40f429b747
+      default: quay.io/redhat-user-workloads/rpm-build-pipeline-tenant/environment:eb178e6f7b63152c384dc3e112dd2068000d56aa
```

Comment:
I think we already use the latest image.  No need to touch this one.

Reply:
Done!